client/components/script-loader.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import CreateLogForm from './create-log-form';
+import { useLoadScript } from '@react-google-maps/api';
+
+const libraries = ['places'];
+
+export default function Script() {
+  const { isLoaded } = useLoadScript({
+    googleMapsApiKey: 'AIzaSyDysHyCDjBRtu-50_S-8fnAedcD8_2V-0Y',
+    libraries
+  });
```

Comment:
You should not commit keys when it's avoidable. Instead use `process.env` and add the environment variable name to Webpack. Put the api key in your .env file.

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs